default/stringtables/en.txt

```diff
@@ -1147,6 +1152,18 @@ Gas Giants Habitable size
 RULE_HABITABLE_SIZE_DESC
 Value used to adjust for [[encyclopedia POPULATION_TITLE_SHORT_DESC]] changes on a planet of this size.
 
+RULE_ALLOW_CONCEDE
+Allow concede
+
+RULE_ALLOW_CONCEDE_DESC
+Allows empire to concede with elimination its assets.
```

Comment:
Allows empires to concede from a game. This eliminates the empire's assets from the universe. The game continues with the remaining players.